### DISCUSSION TITLE
3.4.0 → w-screen h-screen != size-screen

### DISCUSSION THREAD

**Discussion by mv-stns** [tailwindlabs/tailwindcss#12638] [Help]
Hey, so the new update just came out, and I wanted to update my codebase when I realized there is no `size-screen` selector. Of course, extending the config by this:

```js
extend: {
	size: {
		'screen': '100vh',
	}
}
```

is always an option. But I think this should definitely be implemented into tailwinds code.

**Comment by adamwathan** [comment]
Hey! Yeah we didn't add this because that would set both the width and height to `100vh`, when you probably want the width to be `100vw` not `100vh`.

**Reply by mv-stns** [reply]
oh no you're right 🤦🏼‍♂️
